README.md

```diff
@@ -159,3 +163,5 @@ This logging API was designed with the contributors to the Swift on Server commu
 
 - [pitch](https://forums.swift.org/t/logging/16027), [discussion](https://forums.swift.org/t/discussion-server-logging-api/18834), [feedback](https://forums.swift.org/t/feedback-server-logging-api-with-revisions/19375)
 - [log levels](https://forums.swift.org/t/logging-levels-for-swifts-server-side-logging-apis-and-new-os-log-apis/20365)
+
+[api-docs]: https://apple.github.io/swift-log/docs/current/Logging/Structs/Logger.html
```

Comment:
i think just link to `https://apple.github.io/swift-log/`

Reply:
@tomerd I didn't link to that because right now it actually has wrong info ('multiple modules') and I wanted to steer people towards checking `Logger` out first. Happy to change though

<img width="390" alt="Screenshot 2019-03-18 at 6 37 43 pm" src="https://user-images.githubusercontent.com/624238/54554680-ed8aa600-49ac-11e9-9963-7f9dbd9ec82d.png">

Reply:
@tomerd I'll merge this one so I can message the forums/twitter :). Let's fix in follow up PR.